pkg/dockerbuild/tarcopy.go

```diff
@@ -93,13 +96,45 @@ func (tc *tarCopier) CopyData(spec *ImageSpec) error {
 	return nil
 }
 
+// shouldInclude returns true if the path should be included in the tar.
+func shouldInclude(desc *dirCopyDesc, path HostPath) bool {
+	if len(desc.IncludeSrcPaths) == 0 {
+		return true
+	}
```

Comment:
This behavior is a bit strange to me; why do we always return `true` if the list is empty?

Reply:
good catch, I think this is a remnant from some refactoring, let me have a look, I think we can just remove that